gadget/gadget.go

```diff
@@ -406,7 +406,7 @@ func AllDiskVolumeDeviceTraits(allLaidOutVols map[string]*LaidOutVolume, optsPer
 			// at the expected locations, but that is probably fragile and very
 			// non-performant.
 
-			if !vs.IsPartition() {
+			if !vs.VolumeStructure.IsPartition() {
```

Comment:
judging from number of occurences is seems it would make sense for this one and also HasFileystem to also exist on LaidOutStructure

Reply:
Added now